lib/index.ts

```diff
@@ -158,15 +153,16 @@ class PWMetrics {
     }
   }
 
-  async launchChrome(): Promise<ChromeLauncher> {
+  async launchChrome(): Promise<LaunchedChrome|Error> {
     try {
-      this.launcher = new ChromeLauncher();
-      await this.launcher.isDebuggerReady();
-      return this.launcher;
-    } catch(e) {
       console.log(messages.getMessage('LAUNCHING_CHROME'));
-      await this.launcher.run();
-      return this.launcher;
+      return this.launcher = await launch({
+        port: this.flags.port,
```

Comment:
If you use a dynamic port (no port flag) you need to use the port value that is passed back from launcher to connect with. 

I would be you or lighthouse has 9222 hard coded somewhere that you are not overriding.